docs/tutorials/use-dicom-web-standard-apis-with-curl.md

```diff
@@ -80,7 +81,7 @@ _Details:_
  > * `Content-Type: multipart/related; boundary=ABCD1234`
  > * `Content-Type: multipart/related`

```

Comment:
Do not we need to update headers as well for versioning?

Reply:
such as: 
* Path: ../studies
* Method: POST
* Headers:
    * Accept: application/dicom+json
    * Content-Type: multipart/related; type="application/dicom"
    * Authorization: Bearer {token value}
    * **Api-supported-versions: v{version}**

Reply:
We dont specify the version as header (only specified in the path url). I think you may be referring in the response headers, we specify if the version the user is using is deprecated or not.